exercises/06-redux-saga/src/modules/users/components/users-list.js

```diff
@@ -51,8 +51,8 @@ const mapStateToProps = state => ({
   users: UsersSelectors.getUsersList(state)
 });
 
-const mapDispatchToProps = dispatch => ({
-  addUser: user => dispatch(UsersActions.addUser(user))
-});
+const mapDispatchToProps = {
```

Comment:
this is naturally how people use it in commercial projects. The question is whether we don't want to keep it in an expanded form for educational purposes. Or have it expanded i nthe first exercise and then start using this? thoughts @dinassh @branoholy ?

Reply:
Both options are described in the slides, see pages 33 and 34. The expanded version is not currently used in solutions.